src/utils/cachedFetch.js

```diff
@@ -28,6 +28,11 @@ export const cachedFetch = (url, transformFn, cache = true) => {
         if (cache) {
           requestCache[url] = transformResponse
         }
+
+        // set user located country
+        const GeoIP = xhr.getAllResponseHeaders().match(/GeoIP=(\w+)/)
+        const userCountry = GeoIP && GeoIP[1]
+        localStorage.setItem('user-country', userCountry)
```

Comment:
We have `USER_COUNTRY_STORAGE_KEY` in `useExperimentConfig` and here we are setting 'user-country' directly. Honestly I think it's fine this way... but thought I'd flag it just in case. If we ever need to change this key we have to remember to change it in both places.